src/client/main.js

```diff
@@ -19,7 +22,9 @@ class App extends Component {
 			<BrowserRouter>
```

Comment:
A theme will be material-ui theme will be applied here in future branches

Reply:
Let's do this together on Thursday